Source/Blazorise/Extensions/ColorExtensions.cs

```diff
@@ -14,4 +14,34 @@ public static bool IsNullOrDefault( this Color color )
     {
         return color == null || color == Color.Default;
     }
+
+    /// <summary>
+    /// Determines if the supplied color is defined, i.e. not null or empty.
+    /// </summary>
+    /// <param name="color">Color value to check.</param>
+    /// <returns>True if color is not <c>null</c> or <see cref="Color.Default"/>.</returns>
+    public static bool IsNotNullOrDefault( this Color color )
+    {
+        return color != null && color != Color.Default;
+    }
+
+    /// <summary>
+    /// Determines if the supplied background color is null or empty, i.e. not containing any color.
+    /// </summary>
+    /// <param name="background">Color value to check.</param>
+    /// <returns>True if background color is <c>null</c> or <see cref="Background.Default"/>.</returns>
+    public static bool IsNullOrDefault( this Background background )
+    {
+        return background == null || background == Background.Default;
```

Comment:
Probably can just do `background == Background.Default` since null is never equal to `Background.Default`. 
Otherwise if you want to read exactly,  `is null` if you care.

Reply:
I would leave it explicit just in case we do something with Default in the future.